### PR TITLE
✨ Tabular regression

### LINKED ISSUE #16: Tabular-regression

### PR DESCRIPTION
**Pull request by vvmnnnkv**
Fixes #16

### CODE REVIEW

File: packages/inference/src/tasks/tabluar/tabularRegression.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		data: Record<string, string[]>;
+	};
+};
+
+export interface TabularRegressionOutput {
+	/**
+	 * a list of predicted values for each row
+	 */
+	coordinates: number[];
+}
```
Comment (coyotte508) [inline packages/inference/src/tasks/tabluar/tabularRegression.ts:19]: Should it be `type TabularRegressionOutput = number[]` ?

File: packages/inference/src/tasks/index.ts [hunk 2]
```diff
@@ -32,3 +32,6 @@ export * from "./nlp/zeroShotClassification";
 // Multimodal tasks
 export * from "./multimodal/documentQuestionAnswering";
 export * from "./multimodal/visualQuestionAnswering";
+
+// Tabular tasks
+export * from "./tabluar/tabularRegression";
```
Comment (coyotte508) [inline packages/inference/src/tasks/index.ts:37]: Typo: `tabluar` vs `tabular`

Comment (vvmnnnkv) [reply packages/inference/src/tasks/index.ts]: Thanks! 🙈

**Comment by coyotte508** [comment]
Thanks a bunch :)

**Pull request merged**